src/commands/SinceCommand.ts

```diff
@@ -244,7 +245,26 @@ async function execSinceCommandAux(destinationRoomId: string, event: any, mjolni
                             results.bad += 1;
                         }
                     }
-                    const text_ = `Attempted to ban ${joins.length} users from room ${targetRoomId}, ${results.good} kicked, ${results.bad} failures`;
+                    const text_ = `Attempted to ban ${joins.length} users from room ${targetRoomId}, ${results.good} banned, ${results.bad} failures`;
+                    return {
+                        html: text_,
+                        text: text_
+                    }
+                }
+                case Action.Mute: {
```

Comment:
How is someone going to unmute them, shouldn't they be tracked somewhere? e.g. a `org.matrix.mjolnir.muted_users` state event in the target room?

Reply:
I'm a bit afraid that this could be triggered to exceed a maximal event size.

Or perhaps only tracking the latest operation?

Reply:
Surely the power levels event would exceed the maximum event size first?

Reply:
Very likely, yes.